modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -445,7 +470,8 @@ protected Page<ObjectEntry> getObjectEntries(
 	}
 
 	private ObjectEntry _addObjectEntry(
-			String customStatus, Date date, String title)
+			String customStatus, Date date, String title,
+			long objectDefinitionId)
```

Comment:
Can you sort these parameters?

Reply:
What do you think about overloading this method, so you don't have to change all the invocations?

Reply:
I fear that if we don't change all the invocations we might run into the same issue of concurrency

Reply:
My idea is to overload this method, so you would have these two private methods:

`private ObjectEntry _addObjectEntry(
			String customStatus, Date date, String title) {
			
			this._addObjectEntry(customStatus, date, title, _objectDefinition.getObjectDefinitionId())`
			
`private ObjectEntry _addObjectEntry(
			String customStatus, Date date, String title,
			long objectDefinitionId)`
			
The existing method calls the new one passing the default objectDefinitionId value. For our current tests we don't need to pass a different objectDefinitionId in these calls, so we have two options for developers, one using the default value and another if they want to pass a value other than that. Does it make sense?

Reply:
It's much better now, but I think you forgot to sort the parameters.

Reply:
I did, my bad!